continuous_integration/environment-3.12.yaml

```diff
@@ -23,13 +23,15 @@ dependencies:
   - moto<5
   # Optional dependencies
   - mimesis
-  - numpy
+  - numpy >=2
   - pandas
-  - numba
+  # Temporarily removing to allow `numpy >=2` to be installed
+  # - numba
   - flask
   - fastparquet
   - h5py
-  - pytables
+  # Temporarily removing to allow `numpy >=2` to be installed
+  # - pytables
```

Comment:
PyTables will soon be installable with NumPy 2. Just working out some minor kinks: https://github.com/PyTables/PyTables/issues/1200